### DISCUSSION TITLE
Confusing pricing structure

### DISCUSSION THREAD

**Discussion by cupcakearmy** [photoprism/photoprism#3600] [General]
Hey there, just wanted to leave some feedback.

I was searching for an alternative to apple/google photos and basically there is immich and photoprism.
At first, photoprism looked really nice, and I also don't mind paying a bit a month for development.

The thing that really put me off was the incredibly confusing pricing scheme.
Person vs Team, 3 options in each category, what is included where?
I think you should simplify that a lot.
I'm sure some smart person though of it, and I'm probably wrong, but just in my case you lost me as a customer to confusion.

That said, wish you all the best!
Grüße nach Berlin

**Comment by lastzero** [comment]
Teams is aimed at businesses, so I don't know why you're comparing it to a home user solution. Apart from that, most of the features are free and our website offers a complete overview. We need to offer different membership levels because not everyone has the same budget and needs, for example when it comes to technical support.

**Reply by cupcakearmy** [reply]
Just wanted to leave my personal journey as feedback.
As I stated above, I'm sure smart people though of it.
I was taking into consideration business because it was not evident to me how they differ, since both options have multi user support.

**Reply by lastzero** [reply]
As noted on the [Teams and Organizations](https://www.photoprism.app/teams#compare) pricing page, our business plans include a commercial license and an optional SLA for support:

> A key difference between the [Public License](https://docs.photoprism.app/license/agpl) and our [Pro License Agreement](https://www.photoprism.app/pro/license) is that you get access to additional support and configuration options, as well as the right to customize functionality to your needs without having to publicly disclose your changes.

As a result, they are priced significantly higher than our personal memberships. We recommend that you install our free [Community Edition](https://docs.photoprism.app/getting-started/) before deciding to sign up for a membership.

A complete feature overview can be found at the bottom of each pricing page (it is also linked from many other pages, e.g. our FAQs):

- https://www.photoprism.app/editions#compare
